uktrade/data-hub-frontend
# Make specific programme field mandatory when project is in active stage

## Description of change

This change makes the 'Specific investment programme' field mandatory when moving an investment project from the 'Active' stage to the 'Verify Win' stage.

## Test instructions

1. When viewing the details page of an investment project that is in the 'Active' stage and 'Specific investment programme' has not been entered in the 'Edit Summary' page- 'To move to the Verify win stage complete the following: • Specific investment programme' should be visible in the blue box above the 'Investment project summary' section of the page
![Screenshot 2024-06-04 at 13 31 02](https://github.com/uktrade/data-hub-frontend/assets/74198488/7c4b9e37-637e-422d-a6bd-f42b7bbd4141)

2. When viewing the details page of an investment project that is in the 'Active' stage and 'Specific investment programme' has been selected and all other mandatory fields have been completed- 'Before you move to the Verify win stage: Upload any evidence documents Only the Project Manager or Project Assurance Adviser can move the project to the Verify win stage.' should be visible in the blue box above the 'Investment project summary' section of the page
![Screenshot 2024-06-04 at 13 37 37](https://github.com/uktrade/data-hub-frontend/assets/74198488/ca8859d1-9603-42ee-afe7-73d7e27a9dde)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
